MAPL_cfio/ESMF_CFIOEosMod.F90

```diff
@@ -57,43 +62,37 @@ module ESMF_CFIOEosMod
       contains
 
 !------------------------------------------------------------------------------
-!BOP
-! !ROUTINE: ESMF_CFIOEosFileCreate -- Create a CFIO output file with meta data
-
-! !INTERFACE:
-      subroutine ESMF_CFIOEosFileCreate (cfio, rc)
+!>
+! `ESMF_CFIOEosFileCreate` -- Create a CFIO output file with meta data
 !
-! !ARGUMENTS:
+      subroutine ESMF_CFIOEosFileCreate (cfio, rc)
 !
 ! !INPUT PARAMETERS:
 !
```

Comment:
What purpose do these blocks serve.   Fortran already indicates IN/OUT, and ... here there are only 2 arguments.  If we have procedures with long lists of arguments, it may be worth some comment that separates them.  But even then, 3 lines?